### ISSUE TITLE
API parameter to filter events that are in progress

### ISSUE THREAD

**Issue by sergeknystautas** [blakeblackshear/frigate#4971]
**Describe what you are trying to accomplish and why in non technical terms**
The API exposes a count of how many objects that are detected in a zone or camera, for example there are 3 cars.  However you must query all events and filter on the client side to find those 3 cars.

**Describe the solution you'd like**
I would like to add a parameter to the API to have the filtering for in progress events done on the server side and return the specific events for easier use.

**Describe alternatives you've considered**
You could rely on notifications through MQTT of what objects are active, but that is challenging to identify which events are no longer active.  You can query all of the events to understand which are still in progress, but some events can last for a long time so you have to continue going through until you find them.

**Comment by sergeknystautas** [comment]
I created a simple fork that updated the docs and added the API parameter to then have a default and add the parameter to the sqlite parameter in https://github.com/sergeknystautas/frigate-in-progress.

Unfortunately I had a lot of issues getting this to build and finally was able to confirm make worked, but I was unable to do this on the machine that I run frigate on, so I could not validate this.  I hope this gives someone more knowledgable can either confirm this is right or this is a first step towards a solution to this.

**Comment by blakeblackshear** [comment]
Can you tell us more about what you are actually trying to accomplish? Why are you trying to get this information?

**Comment by sergeknystautas** [comment]
This PR started based on a reddit thread I raised here: https://www.reddit.com/r/frigate_nvr/comments/10225bf/how_to_best_see_multiple_objects/

TL;DR - the simplest example I'd like to see is what cars are parked on the street in front of my house.  I have a zone for a street area, and frigate does a good job telling me there are 0 to 4 cars there, but it currently only shares 1 car image.  This would let me quickly get multiple images to match what's shared in the count.

**Comment by sergeknystautas** [comment]
Turned my fork into a PR: #5013

**Comment by NickM-27** [comment]
Already merged
